lab_4_fill_words_by_ngrams/main.py

```diff
@@ -98,6 +135,32 @@ def run(self, seq_len: int, prompt: str) -> str:  # type: ignore
                 or if sequence has inappropriate length,
                 or if methods used return None.
         """
+        if not (isinstance(seq_len, int) and seq_len > 0
+                and isinstance(prompt, str) and prompt):
+            raise ValueError
+        encoded_text = self._word_processor.encode(prompt)
+        if not encoded_text:
+            raise ValueError
+        for i in range(seq_len):
```

Comment:
if the variable is not used anywhere, name it `_`

Reply:
Fixed